### ISSUE TITLE
Unable to start 

### ISSUE THREAD

**Issue by andyoldboy** [WeeJeWel/wg-easy#293]
Hello! I am using yaml to launch, and I bumped into this issue:

```
[+] Running 1/1
 ⠿ Container wg-easy  Created                                                                                                                                                              0.1s
Attaching to wg-easy
wg-easy  | 2022-06-19T16:23:27.408Z Server Listening on http://0.0.0.0:51821
wg-easy  | 2022-06-19T16:23:27.410Z WireGuard Loading configuration...
wg-easy  | $ wg genkey
wg-easy  | $ echo ***hidden*** | wg pubkey
wg-easy  | 2022-06-19T16:23:27.429Z WireGuard Configuration generated.
wg-easy  | 2022-06-19T16:23:27.430Z WireGuard Config saving...
wg-easy  | 2022-06-19T16:23:27.432Z WireGuard Config saved.
wg-easy  | $ wg-quick down wg0
wg-easy  | $ wg-quick up wg0
wg-easy  | Error: WireGuard exited with the error: Cannot find device "wg0"
wg-easy  | This usually means that your host's kernel does not support WireGuard!
wg-easy  |     at /app/lib/WireGuard.js:63:19
wg-easy  |     at processTicksAndRejections (internal/process/task_queues.js:95:5)
wg-easy  |     at async /app/lib/WireGuard.js:61:9
wg-easy exited with code 1
wg-easy  | 2022-06-19T16:23:28.602Z Server Listening on http://0.0.0.0:51821
wg-easy  | 2022-06-19T16:23:28.604Z WireGuard Loading configuration...
wg-easy  | 2022-06-19T16:23:28.605Z WireGuard Configuration loaded.
wg-easy  | 2022-06-19T16:23:28.606Z WireGuard Config saving...
wg-easy  | 2022-06-19T16:23:28.607Z WireGuard Config saved.
wg-easy  | $ wg-quick down wg0
wg-easy  | $ wg-quick up wg0
wg-easy  | Error: WireGuard exited with the error: Cannot find device "wg0"
wg-easy  | This usually means that your host's kernel does not support WireGuard!
wg-easy  |     at /app/lib/WireGuard.js:63:19
wg-easy  |     at async /app/lib/WireGuard.js:61:9
wg-easy exited with code 1
```

Could you help me with that?

**Comment by WeeJeWel** [comment]
Read.

**Comment by eandersons** [comment]
> `This usually means that your host's kernel does not support WireGuard!`

says it all.

What is your server kernel's version?

You should try to update your server's kernel (WireGueard is included in kernels [since version 5.6](https://duo.com/decipher/wireguard-vpn-added-to-linux-kernel)). If that is not possible, then you could try to [install-`wireguard-tools`](https://www.wireguard.com/install/) or  [build WireGuard kernel module](https://www.wireguard.com/compilation/#kernel-requirements) manually.
